### ISSUE TITLE
How to use the pretrained model?

### ISSUE THREAD

**Issue by yjsb** [uci-cbcl/NoduleNet#42]
I downloaded the file '200.ckpt' on Google Drive, and I changed its name to '001.ckpt' in my project. I trained the model for 50 epoches on fold 3, but I get a poor mean dice(exclude fn); 0.74, which in the paper is 0.83. So it maybe something wrong in my training procedure, can u help me, pls?

**Comment by yjsb** [comment]
The learning rate I used in my training procedure was 0.0001, I think it is small enough.

**Comment by yjsb** [comment]
I also tested it on 0_val.csv, and got the mean dice(exclude fn): 0.742. The model is download file '200.ckpt'.
